lamindb/_context.py

```diff
@@ -88,9 +101,10 @@ def reinitialize_notebook(
         new_version = "0"
     metadata["version"] = new_version
 
-    transform = Transform(
-        stem_id=new_id, version=new_version, type=TransformType.notebook
-    )
+    if transform.stem_id != new_id or transform.version != new_version:
```

Comment:
Could you add an extensive comment to this line? I don't fully understand what's going on!